aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -6,6 +6,10 @@
     "Arn": {
       "type": "string",
       "pattern": "^arn:aws(-(cn|gov))?:[a-z-]+:(([a-z]+-)+[0-9]+):([0-9]{12}):[^.]+$"
+    },
+    "ArnIam": {
+      "type": "string",
+      "pattern": "^arn:aws(-(cn|gov))?:iam::([0-9]{12}):[^.]+$"
```

Comment:
[The GovCloud partition is incorrect and the isolated partitions are missing in the current Arn definitions](https://github.com/aws-cloudformation/cloudformation-cli/issues/374#issuecomment-620921094)

Reply:
Looking into it in https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeguru-profiler/pull/19